eos/parameters/hadron-properties.yaml

```diff
@@ -5,177 +5,233 @@ groups:
   # Pseudoscalar Meson Masses
   # {{{
   - title: 'Pseudoscalar Meson Masses'
-    description: ''
+    description: 'Last update: December 2020'
```

Comment:
I would not do that. The web page display for which version of EOS this was done.

Reply:
I agree, I will remove this